Sources/SuperValidator/Validators/Phone.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  File.swift
+//  
+//
+//  Created by christopher.mienarto on 23/03/22.
+//
+
+import Foundation
+
+public enum PhoneFormat {
+    case international
+    case nanp
+    case epp
+}
+
+extension SuperValidator.Option {
+    /// Phone
+    public struct Phone {
+        public let phoneFormatType: PhoneFormat
+        public let specifiedCountryCode: [String]
+
+        public init(
+            phoneFormatType: PhoneFormat = .international,
+            specifiedCountryCode: [String] = []
+        ) {
+            self.phoneFormatType = phoneFormatType
+            self.specifiedCountryCode = specifiedCountryCode
+        }
+    }
+}
+
+// MARK: - Error
+extension SuperValidator {
+    public enum PhoneError: Error, LocalizedError {
+        case invalidCountryCode
+        case invalidPhone
+        
+        public var errorDescription: String? {
+            switch self {
+            case .invalidCountryCode:
+                return nil
```

Comment:
No error description expected?

Reply:
Because it will make more customizable

Reply:
I think this one you can create 2 different messages.

"Please enter a valid phone number"
"Country code is invalid"

Reply:
I already added several error